tsconfig.json

```diff
@@ -16,7 +16,13 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "experimentalDecorators": true,
-    "emitDecoratorMetadata": true
+    "emitDecoratorMetadata": true,
+    "baseUrl": ".",
+    "paths": {
+      "@util/*": ["src/util/*"],
```

Comment:
love the aliases - good stuff 👍 (nitpick) please confirm that we can also run `npm run lint-fix` and `npm run build` but this looks great!

Reply:
Confirmed it works as expected.